example/src/SentimentAnalysisPage.js

```diff
@@ -293,7 +298,36 @@ export default class SentimentAnalysisPage extends React.Component {
                 rating: newRating.toFixed(1),
               })
             }
-          />
+          /> */}
+
+            {<Rating 
+              //defaultValue={2}
+              value={this.state.rating} 
+
+              onChange={(event, newRating) =>
+                //this.setState({
+                //  rating: newRating.toFixed(1),
+                //})
+                console.log(event)
+              }
```

Comment:
remove this whole onChange prop. it's not doing anything useful. We also generally don't want console logs in deployed environments.